recipes/minitrace/all/conanfile.py

```diff
@@ -0,0 +1,66 @@
+from conan import ConanFile, tools
+from conan.tools.cmake import CMakeToolchain, CMake
+from conan.tools.layout import cmake_layout
```

Comment:
```suggestion
from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout
```
https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#the-layout-method